src/components/dashboard/data/utils.js

```diff
@@ -0,0 +1,11 @@
+import { ASSIGNMENT_TYPES } from '../../enterprise-user-subsidy/enterprise-offers/data/constants';
+
+export default function getActiveAssignments(assignments) {
+  const filteredAssignments = assignments?.filter((assignment) => assignment?.state === ASSIGNMENT_TYPES.ALLOCATED
```

Comment:
nit: the conditional could be simplified (i.e., removes the chained `||`) as something like:

```jsx
.filter((assignment) => [ASSIGNMENT_TYPES.ALLOCATED, ASSIGNMENT_TYPES.CANCELLED].includes(state))  
```